src/background.js

```diff
@@ -422,47 +422,33 @@ const rollout = {
     let skipHeuristicsCheck = await rollout.getSetting(DOH_SKIP_HEURISTICS_PREF, false);
```

Comment:
This line is not needed, or we just want to log the value. Anyway the comment is wrong.